app/mailers/admin_mailer.rb

```diff
@@ -0,0 +1,35 @@
+class AdminMailer < ActionMailer::Base
+  LOG_TAG = "AdminMailer"
```

Comment:
What is LOG_TAG ?

Reply:
I copied the pattern from the other mailers.  It looks like the purpose is to specify in the log which class is creating the log message.

Reply:
It's my little invention.  The wrpapper class the Platform uses lets you have a "meta" field in the log json.  You can out any fields you want into that meta field.  So I started putting the name of the class or execution path in there.  It's an idea from the Android Studio environment -- when you want to only filter log messages that are coming out of a specific process.  Like, "show me all the debug statements that registering the user is making".